src/Achievements/AchievementData.json

```diff
@@ -1,6 +1,5 @@
 {
-  "note": "***** Generated from a script, overwritten by steam achievements data *****",
-  "fetchedOn": 1641517584274,
+  "note": "Originally generated by a script using Steam achievement data. Going forward, must be edited manually.",
```

Comment:
Were you able to run this script? I don't *think* they've changed on Steam since this PR was originally made, but I'm not 100% sure.